steer_drive_controller/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(steer_drive_controller)
+
+find_package(catkin REQUIRED COMPONENTS
+  controller_interface
+  urdf
+  realtime_tools
+  tf
+  nav_msgs
+  roscpp
+  angles
+  control_toolbox
+  gazebo_ros_control
+  hardware_interface
+  joint_limits_interface
+)
+
+find_package(gazebo REQUIRED)
+
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
+  CATKIN_DEPENDS
+  roscpp
```

Comment:
please sort these dependencies
(in vim you can do it by selecting these lines in visual mode and `:sort` )

Reply:
agreed